src/LibPQ.jl

```diff
@@ -26,37 +33,36 @@ const LOGGER = getlogger(@__MODULE__)
 
 function __init__()
     INTERVAL_REGEX[] = _interval_regex()
-    Memento.register(LOGGER)
+    return Memento.register(LOGGER)
 end
 
 # Docstring template for types using DocStringExtensions
-@template TYPES =
-    """
-        $(TYPEDEF)
+@template TYPES = """
+                      $(TYPEDEF)
 
-    $(DOCSTRING)
+                  $(DOCSTRING)
 
-    ## Fields:
+                  ## Fields:
 
-    $(TYPEDFIELDS)
-    """
+                  $(TYPEDFIELDS)
+                  """
 
 include(joinpath(@__DIR__, "utils.jl"))
 
 module libpq_c
-    export Oid
+export Oid
```

Comment:
Do you think these changes are correct for a nested module?

Reply:
since we don't indent after `module` as a general rule, i slightly prefer to never do it, than to do it if it's a submodule.

i've seen both styles used, and generally prefer "don't indent", but don't  feel strongly

Reply:
I think if we don't require indentation, we should require a blank line after the module declaration and a blank line before the `end`.

Reply:
https://github.com/invenia/BlueStyle/issues/75